### PR TITLE
Offline Map Integrated ( Tables and Graphs updated)

### PR DESCRIPTION
**Pull request by hridaydutta123**

### CODE REVIEW

**Comment by ghoshbishakh** [comment]
please try to resolve the merge conflicts and then push again.

the process is:
checkout to a new branch and keep it untouched.
then go back to the previous branch and do `git pull upstream master`
resolve whatever merge conflicts arise
then git push to your repo and this PR will be autometically updated

**Comment by hridaydutta123** [comment]
Resolved Merge Conflicts.

**Comment by ghoshbishakh** [comment]
@hridaydutta123 I think there are some map tiles in the repo? Can you remove them? And add a gitignore entry to ignore map tiles

**Comment by ghoshbishakh** [comment]
@hridaydutta123 delete the existing tiles. karon ogulo to code noi. change hote pare

**Comment by hridaydutta123** [comment]
Added tiles/ and log to gitignore....

**Comment by ghoshbishakh** [comment]
@hridaydutta123 yep. kichu existing tiles ache na? ogulo remove kore dao repo theke

**Comment by ghoshbishakh** [comment]
@hridaydutta123 okey nei mone hochhe

**Comment by hridaydutta123** [comment]
All done.. :-)

File: hs_err_pid21365.log [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+ 0x767ad90c: istate->_method       = java.lang.Object.<init>()V
+ 0x767ad910: istate->_mdx          = 0x00000000
+ 0x767ad914: istate->_stack        = 0x767ad8f8
+ 0x767ad918: istate->_msg          = 0x00000002
+ 0x767ad91c: istate->_result       = 0x00000000
+ 0x767ad920: (istate->_result)     = 0x767ad8e4
+ 0x767ad924: (istate->_result)     = 0x767ad8e4
+ 0x767ad928: istate->_prev_link    = 0x00000000
+ 0x767ad92c: istate->_oop_temp     = 0x00000000
+ 0x767ad930: istate->_stack_base   = 0x767ad8fc
```
Comment (ghoshbishakh) [inline hs_err_pid21365.log]: @hridaydutta123 what is this? remove the file if unnecessary.

**Comment by ghoshbishakh** [comment]
@hridaydutta123 LGTM. Thanks :smile:

**Pull request merged**